migrators/ternmigrator/tern_test.go

```diff
@@ -0,0 +1,71 @@
+package ternmigrator_test
+
+import (
+	"embed"
+	"fmt"
+	"io/fs"
+	"testing"
+
+	_ "github.com/jackc/pgx/v5/stdlib" // "pgx" driver
+	"github.com/peterldowns/testy/assert"
+	"github.com/peterldowns/testy/check"
+	"golang.org/x/net/context"
+
+	"github.com/peterldowns/pgtestdb"
+	"github.com/peterldowns/pgtestdb/migrators/ternmigrator"
+)
+
+//go:embed migrations/*.sql
+var exampleFS embed.FS
+
+func TestTernMigrator(t *testing.T) {
+	t.Parallel()
+
+	for _, test := range []struct {
+		name string
+		fs   fs.FS
+	}{
+		{name: "FromDisk"},
+		{
+			name: "FromFS",
+			fs:   exampleFS,
+		},
+	} {
```

Comment:
Same preference for two separate testcases rather than a two-item for loop. Similarly not a blocker, just a request.